common/app/model/dotcomrendering/MostPopular.scala

```diff
@@ -187,8 +186,24 @@ object MostPopularGeoResponse {
 // MostPopularNx2 was introduced to replace the less flexible [common] MostPopular
 // which is heavily relying on pressed.PressedContent
 // because we want to be able to create MostPopularNx2 from trails coming from the DeeplyReadAgent
-case class MostPopularNx2(heading: String, section: String, trails: Seq[OnwardItemNx2])
+case class MostPopularNx2(heading: String, section: String, trails: Seq[OnwardItem])
 
 object MostPopularNx2 {
   implicit val mostPopularNx2Writes = Json.writes[MostPopularNx2]
 }
+
+object OnwardsUtils {
+
+  def determinePillar(pillar: Option[Pillar]): String = {
+    pillar.map { pillar => correctPillar(pillar.toString.toLowerCase()) }.getOrElse("news")
+  }
+
+  def correctPillar(pillar: String): String = {
+    if (pillar == "arts") {
+      "culture"
+    } else {
+      pillar
+    }
+  }
```

Comment:
I think combining these 2 methods into a single pattern match would be more idiomatic scala/cleaner? e.g.

```
def normalisePillar(pillar: Option[Pillar]): String = pillar.toString.toLowerCase() match {
  Some("arts") => "arts"
  Some(pillar) => pillar
  None(_) => "news"
}
```

Reply:
Thank you for your help on this!